.github/workflows/exporter-ci.yml

```diff
@@ -0,0 +1,103 @@
+name: Exporter CI
```

Comment:
for all workflow, I would like to have an higher lever description to see what is the intent of the workflow

like. Build and test XX foo .

Reply:
I will add some comments with a more exhaustive description: I can't use the `name` field, that's meant to be a short name.

Reply:
yeah just add comment